app/controllers/hyrax/downloads_controller.rb

```diff
@@ -0,0 +1,47 @@
+module Hyrax
+  class DownloadsController < ApplicationController
+    include Hyrax::DownloadBehavior
+
+    # Overrides Hydrx::Controller::DownloadBehavior#load_file to load files for visibility metadata-oly and culturally sensitive.
+    # Override this method to change which file is shown.
+    # Loads the file specified by the HTTP parameter `:file`.
+    # If this object does not have a file by that name, return the default file
+    # as returned by {#default_file}
+    # @return [ActiveFedora::File, String, NilClass] Returns the file from the repository or a path to a file on the local file system, if it exists.
+    def load_file
+      file_reference = params[:file]
+      return default_file unless file_reference
+
+      icon = icon_path(params[asset_param_key], true) if file_reference == 'thumbnail'
+      return icon if !icon.nil?
```

Comment:
I'm not sure this needs the nil check. I think you may be able to just say

`return icon if icon`

You might also be able to use a return pattern such as the following for the rest of this method (just a thought):

```
result =
  if condition
    x
  else
    y
  end
```